apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -114,6 +114,7 @@ public sealed class CloudTrace : IDisposable
             // Create the default values if not set.
             client = client ?? TraceServiceClient.Create();
             config = config ?? TraceConfiguration.Create();
+            traceFallbackPredicate = traceFallbackPredicate ?? TraceDecisionPredicate.Default;
```

Comment:
This is awkward. I suspect we do want the ability to trace even health requests - but *implicitly* disabling the default predicate as soon as you provide a different one feels wrong to me. I suspect most people will want it to be additive (in terms of what it filters), basically.

If TraceDecisionPredicate were public, it would at least be able to do that explicitly, although we should document it carefully (and possibly even provide a method to make it easy to build the compound predicate).